packages/eslint-plugin-pf-codemods/index.js

```diff
@@ -29,10 +29,10 @@ const warningRules = [
 ];
 
 // rules that will run before other rules (move to deprecated?)
-const setupRules = [];
+const setupRules = ["table-update-deprecatedPath"];
 
 // rules that will run after other rules (cleanup imports?)
-const cleanupRules = [];
+const cleanupRules = ["table-rename-TableComposable"];
```

Comment:
do we need to add this to the cleanup rules? i was thinking for this case we'd just need to add the deprecated stuff to the setup.

Reply:
Nice catch, removed